Linter issues with "consider-using-with"

Is this relevant? Should we disable this check?

```
************* Module tests.scripts.utils
tests/scripts/utils.py:43:4: R1732: Consider using 'with' for resource-allocating operations (consider-using-with)
************* Module libcloudforensics.providers.gcp.internal.compute
libcloudforensics/providers/gcp/internal/compute.py:793:4: R1732: Consider using 'with' for resource-allocating operations (consider-using-with)
************* Module libcloudforensics.scripts.utils
libcloudforensics/scripts/utils.py:43:4: R1732: Consider using 'with' for resource-allocating operations (consider-using-with)
```

--- a/libcloudforensics/providers/gcp/internal/compute.py
+++ b/libcloudforensics/providers/gcp/internal/compute.py
@@ -790,16 +790,16 @@ class GoogleComputeInstance(compute_base_resource.GoogleComputeBaseResource):
   def _SshConnection(self) -> None:
     """Create an SSH connection to the virtual machine."""
 
-    devnull = open(os.devnull, 'w')
-    cmd_list = ['gcloud',
-                'compute',
-                '--project',
-                self.project_id,
-                'ssh',
-                '--zone',
-                self.zone,
-                self.name]
-    subprocess.check_call(cmd_list, stderr=devnull)
+    with open(os.devnull, 'w') as devnull:
+      cmd_list = ['gcloud',
+                  'compute',
+                  '--project',
+                  self.project_id,
+                  'ssh',
+                  '--zone',
+                  self.zone,
+                  self.name]
+      subprocess.check_call(cmd_list, stderr=devnull)
 
   def Ssh(self) -> None:
     """Connect to the virtual machine over SSH."""

--- a/libcloudforensics/scripts/utils.py
+++ b/libcloudforensics/scripts/utils.py
@@ -40,10 +40,9 @@ def ReadStartupScript() -> str:
       # Use the provided script
       script_path = os.path.join(
           os.path.dirname(os.path.realpath(__file__)), STARTUP_SCRIPT)
-    startup_script = open(script_path)
-    script = startup_script.read()
-    startup_script.close()
-    return script
+    with open(script_path) as startup_script:
+      script = startup_script.read()
+      return script
   except OSError as exception:
     raise OSError(
         'Could not open/read/close the startup script {0:s}: {1:s}'.format(

--- a/libcloudforensics/scripts/utils.py
+++ b/libcloudforensics/scripts/utils.py
@@ -41,8 +41,7 @@ def ReadStartupScript() -> str:
       script_path = os.path.join(
           os.path.dirname(os.path.realpath(__file__)), STARTUP_SCRIPT)
     with open(script_path) as startup_script:
-      script = startup_script.read()
-      return script
+      return startup_script.read()
   except OSError as exception:
     raise OSError(
         'Could not open/read/close the startup script {0:s}: {1:s}'.format(

--- a/tests/scripts/utils.py
+++ b/tests/scripts/utils.py
@@ -40,13 +40,13 @@ def ReadProjectInfo(keys: List[str]) -> Dict[str, str]:
         '"PROJECT_INFO" environment variable pointing '
         'to your project settings.')
   try:
-    json_file = open(project_info_path)
-    try:
-      project_info = json.load(json_file)  # type: Dict[str, str]
-    except ValueError as exception:
-      raise RuntimeError(
-          'Cannot parse JSON file. {0:s}'.format(str(exception))) from exception
-    json_file.close()
+    with open(project_info_path) as json_file:
+      try:
+        project_info = json.load(json_file)  # type: Dict[str, str]
+      except ValueError as exception:
+        raise RuntimeError(
+            'Cannot parse JSON file. {0:s}'.format(
+                str(exception))) from exception
   except OSError as exception:
     raise OSError(
         'Could not open/close file {0:s}: {1:s}'.format(

